### ISSUE TITLE
Migration 2>3: cache on SQLite does not work

### ISSUE THREAD

**Issue by pcav** [3liz/lizmap-web-client#354]
When migrating from 2.12 to 3.0.1, the layers with a cache are not displayed if the cache is set to SQLite. All well if the cache is on files.

**Comment by mdouchin** [comment]
Here is the log

```
2016-08-19 15:35:23	127.0.0.1	notice	2016-08-19 15:35:23	[8]	Undefined variable: db	/var/www/LIZMAP/lizmap-web-client/lib/jelix/db/jDbPDOConnection.class.php	138

2016-08-19 15:35:23	127.0.0.1	error	2016-08-19 15:35:23	[14]	SQLSTATE[HY000] [14] unable to open database file	/var/www/LIZMAP/lizmap-web-client/lib/jelix/db/jDbPDOConnection.class.php	94
```

It seems something has changed between 2.* & 3.*

**Comment by rldhont** [comment]
@mdouchin it has been fixed isn't it ?

**Comment by laurentj** [comment]
It has been [fixed in Jelix 1.6.10](https://github.com/jelix/jelix/commit/25729724cdf1c498235f1ee44df62f8023994b75) by @mdouchin and so in Lizmap 3.0.3
